docs/modelserving/v1beta1/llm/huggingface/README.md

```diff
@@ -0,0 +1,159 @@
+# Deploy the LLaMA model with Huggingface Serving Runtime
+The Huggingface serving runtime implements a runtime that can serve huggingface transformer based model out of the box. 
+
+In this example, we deploy a LLaMa model from huggingface by running an `InferenceService` with [Hugginface Serving runtime](https://github.com/kserve/kserve/tree/master/python/huggingfaceserver). Based on the performance requirement, you can choose to perform the inference on a more optimized inference engine like triton inference server and [vLLM](https://github.com/vllm-project/vllm) for text generation.
+
+### Serve the huggingface model using KServe python runtime 
+
+=== "Yaml"
+    ```yaml
+    kubectl apply -f - <<EOF
+    apiVersion: serving.kserve.io/v1beta1
+    kind: InferenceService
+    metadata:
+      name: huggingface
+    spec:
+      predictor:
+        containers:
+        - args:
+          - --model_name=llama2
+          - --model_id=meta-llama/Llama-2-7b-chat-hf
+          - --tensor_input_names=input_ids
+          - --disable_vllm
+          image: kserve/huggingfaceserver:latest
+          name: kserve-container
+          resources:
+            limits:
+              cpu: "6"
+              memory: 24Gi
+              nvidia.com/gpu: "1"
+            requests:
+              cpu: "6"
+              memory: 24Gi
+              nvidia.com/gpu: "1"
+    EOF
+    ```
+
+### Serve the huggingface model using triton inference runtime
```

Comment:
Triton inference runtime does not support LLM models like llama2, it requires to use tensorrt-llm backend. For triton we can use BERT as an example and can be moved to the normal serving examples instead of here as the LLM example.